openapi/tests/input_output/input.py

```diff
@@ -46,20 +46,7 @@ class AggregateResultItem(dict):
     # GenStop
 
 
-# GenClass: Asset
-class Asset(CogniteResource):
-    # GenStop
-    def to_pandas(self):
-        pass
```

Comment:
Should we change this to test for example Event instead? So that we still test the `# GenClass` functionality?

Reply:
Yes, I can change that